tensorflow/compiler/mlir/tosa/transforms/legalize_tf.cc

```diff
@@ -738,35 +739,43 @@ LogicalResult ConvertTFShapeOp::matchAndRewrite(
 }
 
 LogicalResult ConvertTFExpandDimsOp::matchAndRewrite(
-    Operation *op, PatternRewriter &rewriter) const {
+    Operation* op, PatternRewriter& rewriter) const {
   auto tf_expanddims_op = cast<TF::ExpandDimsOp>(op);
 
-  auto lowered_op =
+  auto result =
       convertExpandDimsOp(rewriter, op, tf_expanddims_op.getResult(),
                           tf_expanddims_op.input(), tf_expanddims_op.dim());
 
-  TOSA_REPLACE_LOWERED_OP(rewriter, op, lowered_op);
+  if (!result) return failure();
+
```

Comment:
Nit: through all of these: I think it is fine to collapse these blank lines. If you decide to do it, do it to all of them.